tests/data/bundled_suites/_support/_generated/UnitTesterActions.php

```diff
@@ -0,0 +1,547 @@
+<?php  //[STAMP] db3bd0913ded92ae845d337ddebb7e71
```

Comment:
Please delete this file from git.
I've raised a pull request to fix gitignore rules https://github.com/Codeception/Codeception/pull/4439